design-at-cornell/src/components/ButtonStyles.ts

```diff
@@ -18,3 +18,55 @@ export const RectangularButton = styled.div`
     color: black;
   }
 `;
+
+export const ModalButton = styled.div`
+  display: flex;
+  width: 106px;
+  height: 110px;
+  background: white;
+  margin-right: 10px;
+  padding-left: 10px;
+  margin-top: 35px;
+  padding-right: 10px;
+  border-radius: 9px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  cursor: pointer;
+
+  p {
+    width: fit-content;
+    font-weight: bold;
+    font-size: 12px;
+    line-height: 14px;
+    color: black;
+  }
+
+  img {
+    cursor: pointer;
+  }
+`;
+
+export const CUReviewsButton = styled(ModalButton)`
+  box-shadow: 0px 0px 22px #e3bff0;
+  left: 806px;
+  top: 580px;
+`;
+
+export const CourseRosterButton = styled(ModalButton)`
+  box-shadow: 0px 0px 22px #c9e1aa;
+  left: 540px;
+  top: 579px;
+`;
+
+export const CourseSiteButton = styled(ModalButton)`
+  box-shadow: 0px 0px 22px rgba(254, 186, 171, 0.83);
+  left: 673px;
+  top: 579px;
+`;
+
+export const SyllabusButton = styled(ModalButton)`
+  box-shadow: 0px 0px 22px #b1d5ef;
```

Comment:
A good practice is to keep all color values (hex or rgba) in a separate constants file so that components sharing the color can easily access the same value. We have a file `src/constants/colors.ts` that stores all color values. And then to access the colors in styling sheets, you can have `import { colors } from '../constants/colors';` and use `${colors.colorName}` in place of the hex or rgba value.